### ISSUE TITLE
New request. Remote sources?

### ISSUE THREAD

**Issue by pietrondo** [kr15h/ofxPiMapper#69]
Possible new feature:
remote sources..
One pc transmit video and the raspberry output it on the video (in a square, triangle etc).
Possible?
thx

**Comment by kr15h** [comment]
Yes. TCPSyphon source is one of the next things I am going to implement.

**Comment by magdesign** [comment]
This could be done with TCPSyphon or with a more common protocol like RTP/UDP.
Syphon is nice when you have VJ software, RTP is cool because VLC can output it.

**Comment by steve20pro** [comment]
adding to this another request, 
an ability to use an hdmi input as a source 

using a board such as this: http://auvidea.eu/index.php/theme-styles/2014-12-30-22-32-06/b102

**Comment by pietrondo** [comment]
yes. Problem with TCPSyphon. It will work if you have a windows or mac os desktop.
If you have linux you can't output nothing (spout and syphon don't exist on linux)

thx for you software

**Comment by hgshoggins** [comment]
Supporting remote sources would be absolutely wonderful.

If I'm correct, you're already using the FFMpeg API to access the video data, right ? If so, it looks like the possibility to access remote sources is part of it.

For example, I use OMXPlayer with remote sources simply by giving it an SDP file, and I successfully display an H264 RTP (UDP) stream with no effort. Looking into OMXPlayer's code, I couldn't find any evidence of SDP parsing, but found some in FFMPeg's source. As OMXPlayer is using FFMpeg to access the video, I assumed by deduction that it would be feasible in ofxPiMapper.

By the way, ofxPiMapper is just brilliant, thanks !
